Makefile

```diff
@@ -109,8 +109,10 @@ doc: docs/dunst.1 docs/dunst.5 docs/dunstctl.1
 
 # Can't dedup this as we need to explicitly provide the name and title text to
 # pod2man :(
+docs/dunst.1.pod: docs/dunst.1.pod.in
+	${SED} "s|@sysconfdir@|${SYSCONFDIR}|" $< > $@
```

Comment:
Why do you change the sysconfdir in a different way from the other variables?

Reply:
To retain the same configuration formatting as Meson.

Reply:
Oh, i see what you mean.

This is simply because `pod2man` can figure out the section on it's own, as for the title i'll proceed with adding it back.

Reply:
Ah, its because this file in particular gets generated from a .in, the actual man page generation can be seen below for `docs/dunst.1`.

Reply:
Ah okay. That's fine